docs/gettingStarted/customScheme/developCustomNonUniScheme.md

```diff
@@ -0,0 +1,94 @@
+  *Tutorial for adding non-universal scheme* [skip](../developCustomUniScheme)
+
+## Design Principle
+  TODO: Need Oren's input here
```

Comment:
By using a none universal scheme one can attached multiple instances of that scheme to a single dao.
Normally this scheme is more 'simple' than a universal one , as it server a single avatar(dao).
some recommended design principle :
- should include a one time initialize public function , which gets the avatar as its first parameters. this will assist with the common migration process.